lucille-core/src/main/java/com/kmwllc/lucille/core/Connector.java

```diff
@@ -116,7 +116,7 @@ static List<Exception> getConnectorConfigExceptions(Config connectorConfig) {
         }
       }
     } catch (ReflectiveOperationException e) {
-      exceptionList.add(new ConnectorException("Connector class not found: " + e.getMessage()));
+      exceptionList.add(new ConnectorException("Reflective error with Connector: " + e.getMessage()));
```

Comment:
I would say something like "connector class or suitable constructor"... i.e. spell out the likely reflective errors that might occur